.github/DOCUMENTATION_TEMPLATE.md

```diff
@@ -0,0 +1,64 @@
+# How to conttribute to MFTF docs
+
+MFTF documentation is kept within the /docs/ folder in this repo.
+We welcome contributions to the documentation.
+This page describes the process for submitting docs and serves as a template so that docs are written properly.
+
+The contribution workflow for docs is the same as submitting code.
+To contribute to MFTF docs:
+
+1. Make a branch from the [MFTF repo][].
+1. Make edits/additions/deletions as needed.
+1. Submit your PR to the `develop` branch.
+
+Once submitted, it will be reviewed by a member of the documentation team.
+If approved it will be tested and merged.
+If it needs any work, we will inform you.
+
+Any changes to the Table of Contents will need to be made in the regular [devdocs repo][].
+
+## H2 heading - blank line before and after, capitalize first word only
+
+1. Ordered lists all all numbered 1.
+1. The build process will number them correctly.
+1. Single spafce after the number. Blank line before and after list.
+
+### H3
+
+- Unordered lists use dashes.
+- One space after the dash.
+- Blank line before and after list.
+
+## Code Samples
+
+For formatting and code coloring, wrap code samples in the following format.
+Replace the `xml` with the file extension of the code sample type. Use `bash` for command-line text.
+
+```xml
+<xmlSample>
+   ...
+   ...
+</xmlSample>
+```
+
+## Markdown tables
+
+| Header      | Header |
+| ----------- | ----------- |
+| Colume 1 text | Column 2 text|
+| Column 1 text | Column 2 text|
+
+Markdown tables work for simple tables. If you need lists or other complex features within a cell, you may have to use a HTML table.
```

Comment:
```suggestion
Markdown formatting works for simple tables. It does not support multiline content within a cell, or split/merged cells within a row or column.
```
We don't want to use complex tables. The content that requires a complex table should be restructured to avoid it.

Reply:
Thanks.